common/v2/utils/transaction.ts

```diff
@@ -26,7 +26,7 @@ export function fromTxReceiptObj(txReceipt: ITxReceipt): ITxReceipt | undefined
             contractAsset.decimal || DEFAULT_ASSET_DECIMAL
           )
         : fromWei(Wei(hexWeiToString(txReceipt.value._hex)), 'ether').toString(),
-      to: contractAsset ? ERC20.transfer.decodeInput(txReceipt.data)._to : txReceipt.to,
+      to: contractAsset ? ERC20.transfer.decodeInput(txReceipt.data)._to : txReceipt.to || '0x',
```

Comment:
It seems dangerous to default to '0x' for all transaction formatting.
Can you please explain?

Reply:
Usually, we have `to` parameter when sending a transaction but when we deploy a contract there is no `to` parameter so I just used a creation address (0x) as a placeholder. We can change it to some other address. Please tell me to which. Thanks

Reply:
Since we don't need the `to` param for a contract deployment it would be better to make it explicit and remove it from the object.
If the view requires a `to` field, we can disable the field when displaying a deployment receipt.
Would this work for you?

Reply:
The only problem is in transaction list in dashboard. If saved transaction doesn't have `to` field, the app then crashes when trying to display transaction list. I can probably do some check there and just show blank address for contracts deplyoments.

Reply:
Sounds good! It would also be more reassuring for the user than seeing a tx to 0x in his history.

Reply:
Fixed